examples/mm/stable_diffusion/conf/draftp_sdxl.yaml

```diff
@@ -0,0 +1,281 @@
+name: sdxl_draftp_train
```

Comment:
@rohitrango Do you mind leaving a comment like
```suggestion

# TODO(@rohitrango): Remove after SD/SDXL APIs merged in NeMo so only one config is needed in Aligner
```
This way we won't forget that to deduplicate this code once Nemo APIs are merged